app/themes/skins/FormFieldOwnSkin.scss

```diff
@@ -31,6 +31,14 @@
     border-radius: 8px;
 
     margin-top: -17px;
+    background-color: transparent;
+  }
+
```

Comment:
I think it's better of add some comment about the trick.
basically what problem we are fixing by this trick, so it will help others understand in future.

Reply:
Agree, commited.